### ISSUE TITLE
the Directory plugin is not working in DS Langflow

### ISSUE THREAD

**Issue by fanimali** [langflow-ai/langflow#4396]
### Bug Description

I am trying to read files from a directory, using the Data>Directory plugin, in Datastax Astra Langflow.

The Directory plugin doesn't do anything when the user clicks on the Path box.

In theory, one expects that a filw browser opens and allows the user to choose a directory on the local filesystem, like it is the OSS Langflow.

Can you please comment if this plugin should work like in the OSS Langflow or not?
![image](https://github.com/user-attachments/assets/7d197a33-e485-4a2c-b670-00b5947a9ae1)

### Reproduction

1. open Langflow on Datastax Astra portal.
2. open a generic flow or create a new one
3. drag in the Data>Directory plugin and try to read files from a directory.

### Expected behavior

the Directory plugin should allow users to import files from a dir

### Operating System

ASTRA

### Langflow Version

1.0.9-post1

### Python Version

None

### Screenshot

![image](https://github.com/user-attachments/assets/ac23809f-2300-4433-a9ba-5194b271adf8)

**Comment by erichare** [comment]
@fanimali this is a great point - I'm working on improvements to the File Component, that allow uploading a zip and recursively parsing the zip. I think this is a good workaround right now (this is already merged into GitHub). 

The directory component will likely be retired... the basic issue is it requires a path, but the path is a string - and relative to the machine Langflow is running on. As you noted. The file component will take its place. Now, you're right however that the ideal UI would not be uploading a zip, but allowing selection of a directory. With that in mind, there still needs to be UI work to allow that - the zip solution is somewhat of a workaround.

TLDR:

1. Current workaround - upload a zip with the new file component, and all files will be recursively processed.
2. Pending improvement - allow multiselect / uploading of a directory in the file component

Once #2 is complete, i think we can deprecate / remove the directory component.
